src/User32.Desktop/User32+SafeDCHandle.cs

```diff
@@ -26,8 +26,10 @@ public class SafeDCHandle : SafeHandle
             /// </summary>
             /// <param name="hWnd">The HWND this handle is associated with and must be released with.</param>
             /// <param name="hDC">The handle to the DC.</param>
-            public SafeDCHandle(IntPtr hWnd, IntPtr hDC)
-                : base(IntPtr.Zero, true)
+            /// <param name="ownsHandle"><see langword="true"/> to reliably release the handle during the finalization
+            /// phase; <see langword="false"/> to prevent reliable release.</param>
```

Comment:
I agree that it isn't clear, I copy pasted it from another `SafeHandle` we already have but it seem to have been taken from the documentation of `SafeHandle` itself :

![2016-02-25 10_01_55-safehandle constructor system runtime interopservices](https://cloud.githubusercontent.com/assets/131878/13314709/d5d84494-dba6-11e5-8cdf-3367c37de005.png)